.travis.yml

```diff
@@ -64,6 +64,9 @@ env:
 before_install:
   - sudo apt-get update
   - sudo apt-get install -y libc6-dev-i386 g++-multilib lib32stdc++6
+  - wget https://cmake.org/files/v3.17/cmake-3.17.0-Linux-x86_64.sh && mkdir /opt/cmake && sh cmake-3.17.0-Linux-x86_64.sh --prefix=/opt/cmake --skip-license && sudo ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake && rm cmake-3.17.0-Linux-x86_64.sh
```

Comment:
Please update in the same way Dockerfile for Ubuntu

Reply:
BTW why do we use cmake-3.17 here when minimum required version is 3013?

Reply:
It's already in Ubuntu Dockerfile, I've borrowed it from there. 3.17 is just because it's the latest.